reana_server/cli.py

```diff
@@ -271,6 +272,50 @@ def import_users(ctx, admin_access_token, file_):
             fg='red', err=True)
 
 
+@reana_users.group(
+    'token',
+    help='All interaction related to user tokens on REANA cloud.')
+def reana_users_token():
+    """Manage REANA user tokens."""
+
+
+@reana_users_token.command('grant')
+@click.option(
+    '--id',
+    help='The id of the user.')
+@click.option(
+    '--admin-access-token',
+    default=os.environ.get('REANA_ADMIN_ACCESS_TOKEN', None),
+    help='The access token of an administrator.')
+def token_grant(id, admin_access_token):
```

Comment:
What about allowing to grant by email address? The email is supposed to be unique in the database so it shouldn't be a problem and it would save us the step of searching the user id. If it's too much work it can be ticketized and done later IMO.

Same for revoking.